src/extract.ts

```diff
@@ -304,6 +304,69 @@ function extractGoogleCppResult(output: string): BenchmarkResult[] {
     });
 }
 
+function extractCatch2Result(output: string): BenchmarkResult[] {
+    const lines = output.split('\n');
+
+    const ret = [];
+    // Example:
+    //   test bench_fib_20 ... bench:      37,174 ns/iter (+/- 7,527)
+    const reTestCaseStart = /^(benchmark name) +(samples) +(iterations) +(estimated)/;
```

Comment:
Please remove redundant '(' and ')' since I want to keep regex as simple as possible.

```suggestion
    const reTestCaseStart = /^benchmark name +samples +iterations +estimated/;
```

Reply:
Fixed.